### PR TITLE
Changed my affiliation to Swansea University and added acknowledgements

### PR DESCRIPTION
**Pull request by rosskynch**
After thinking about it, I thought it'd be best to put the university I did most of the work at. Also added the funding body and grant to the acknowledgements.

### CODE REVIEW

**Comment by bangerth** [comment]
That sounds reasonable. I just merged another PR that now conflicts with yours. Can you rebase?

**Comment by bangerth** [comment]
By the way, I'll make the same comment as in #25: You're now one of the authors of the paper, equal with all others. Any edits you might have would be greatly appreciated -- in fact, as one of the few native speakers of the language we write in, you're probably better positioned to find language issues than the rest of us :-)

**Comment by rosskynch** [comment]
Ok, have rebased.

No problem - I'll give the rest of the paper a review and put in a new PR for anything I spot.

File: 9.1/paper.tex [hunk 1]
```diff
@@ -856,6 +857,10 @@ \section{Acknowledgments}
 ``Performance tuning of high-order discontinuous Galerkin solvers for
 SuperMUC-NG''.
 
+R.~M.~Kynch was supported by the Engineering and Physical Science Research
+Council (EPSRC) UK through grant EP/K023950/1 while working at Swansea
+University, UK 2013-2015 where the majority of his contribution was
+undertaken.
```
Comment (kronbichler) [inline 9.1/paper.tex]: A final nit-pick: Please fill in an empty line here.

Comment (rosskynch) [reply 9.1/paper.tex]: Oops, broke that when resolving the conflict. Fixed now.

Comment (kronbichler) [reply 9.1/paper.tex]: Excellent.

**Pull request merged**